api/renderers.py

```diff
@@ -6,16 +6,16 @@
 
 from django.db.models import Manager, QuerySet
 from django.utils import encoding, six
-from rest_framework import relations
+from rest_framework import relations, renderers
 from rest_framework.serializers import (BaseSerializer, ListSerializer,
                                         Serializer)
 from rest_framework.settings import api_settings
 
 import inflection
-from rest_framework_json_api import renderers, utils
+from rest_framework_json_api import utils
 
 
-class JSONRenderer(renderers.JSONRenderer):
+class JSONAPIRenderer(renderers.JSONRenderer):
```

Comment:
Should [meta > count ](https://github.com/CenterForOpenScience/lookit-api/blob/e8eb8e7093d94b33c5a341eccd290beddc0dfdb2/api/renderers.py#L174)information have parallel structure?

<img width="401" alt="screen shot 2017-06-29 at 3 19 25 pm" src="https://user-images.githubusercontent.com/9755598/27706316-29241d3e-5cdf-11e7-87c1-23d1563e3f1c.png">